catalogue/webapp/components/WorkDetails/WorkDetails.js

```diff
@@ -113,9 +116,34 @@ const WorkDetails = ({
     return id.identifierType.id === 'isbn';
   });
 
-  const WorkDetailsSections = [];
+  const [iiifPresentationLocation] = work.items
```

Comment:
We do this quite a lot, might be worth abstracting here:
https://github.com/wellcometrust/wellcomecollection.org/blob/master/common/utils/works.js

Reply:
This branch only use it here, it is duplicated in another branch I'm working on so I'll abstract it in there